Linux/tree-common/init

```diff
@@ -249,6 +249,12 @@ log_begin_msg "Creating default configuration files on rootfs..."
 run mkdir -p "${rootmnt}/etc"
 run touch ${rootmnt}/etc/hostname ${rootmnt}/etc/resolv.conf
 [ -s ${rootmnt}/etc/hostname ] || run sh -ec "echo ${HOSTNAME} > ${rootmnt}/etc/hostname"
+if ! $(cat /etc/hosts | grep -q "127.0.0.1"); then
```

Comment:
Can you add a test to handle the case where we found the address `127.0.1.1` to set the hostname at this line

https://www.debian.org/doc/manuals/debian-reference/ch05.en.html#_the_hostname_resolution